packages/mdc-fab/mdc-fab.scss

```diff
@@ -14,6 +14,7 @@
 
 @import "@material/theme/variables";
 @import "./mixins";
+@import "./variables";
```

Comment:
This isn't actually needed here, is it?

Reply:
oops, forgot to reference it. Updated now.